### PR TITLE
360 update doc links

### PR DESCRIPTION
**Pull request by ahmetoz**
fixes wrong links in docs. 
related to #360.

### CODE REVIEW

File: docs/usage/TYPE_SYNC.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ __Note__ The statistics object contains the processing time of the last batch on
  
 #### More examples of how to use the sync
  
- 1. [Sync from another CTP project as a source](https://github.com/commercetools/commercetools-sync-java/tree/master/src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/types/TypeSyncIT.java).
+ 1. [Sync from another CTP project as a source](https://github.com/commercetools/commercetools-sync-java/tree/master/src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/type/TypeSyncIT.java).
```
Comment (heshamMassoud) [inline docs/usage/TYPE_SYNC.md]: Good catch! but I think the actual package name needs to be changed in this case ;)

Comment (heshamMassoud) [reply docs/usage/TYPE_SYNC.md]: to be consistent with the others

Comment (ahmetoz) [reply docs/usage/TYPE_SYNC.md]: renamed:  https://github.com/commercetools/commercetools-sync-java/pull/361/commits/42b01682ba7e17871d17901bafbbdad66653a686

**Comment by heshamMassoud** [comment]
Should we add a **Documentation** release notes entry for this?

**Pull request merged**